### PR TITLE
Update Battery.md w/ copy edits & typo fixes

### PR DESCRIPTION
**Pull request by payne92**
See diffs -- I made a pass to fix typos, clean up language, copy edit, and clarify various items.

### CODE REVIEW

**Comment by dkisselev** [comment]
Looks good, also solves #740, but could you clarify the ratio bit a little further? Have you also tested that the ratio method works as expected?

In my case here's what I did:

Flew a pack down to 3.75v (around 50%), recorded the Virtual current rating, 1260mah. Then charged the battery, charger reported 1158mah. 

```Charger  / Virtual = Ratio = 1158 / 1260 = ~0.92```

Then I multiplied my ```current_meter_scale``` by that ratio to find my new value: 

``` current_meter_scale * Ratio = 400 * 0.92 = 368```.

I've flown a few packs on this new setting and recorded their virtual readings, but I haven't charged them and haven't been able to test the actual real usage yet.

tl;dr, I think adding a few psuedo-equtions will help clarify the exact process a lot. Feel free to copy my exact example and equations if you'd like.

**Comment by choyado** [comment]
Hi. I'm attempting to calibrate my hardware current sensor and the mathematical logic in the above post does not make sense. Perhaps if using a virtual sensor it works but not on an actual sensor. When using an actual current sensor decreasing the current_meter_scale value actually increases the amp reading in clean flight. See the below scenario:

Lets say I have a current sensor that should output 40mv/A so I initially input 400 in cleanflight
lets say my charger vs virtual ratio is the sames as above. 

Charger / Virtual = Ratio = 1158 / 1260 = ~0.92

Doing as indicated above 

current_meter_scale * Ratio = 400 * 0.92 = 368

The problem is that a value of 368 represents 36.8mv/A where previously it was 40mv/A. Lets say the current sensor is outputting 100mv. With an scale of 400 cleanflight would interpret that as 2.5 amps. Reducing the scale to 368 would increase the amp reading to 2.71.

**Comment by payne92** [comment]
Thanks for your comment; I'm sorry for the delay in reply.

If you have a hardware current sensor, I would use (and trust) that data.  The hardware current sensor should be already calibrated -- no additional work required.

The "virtual" and "charger measurement" methods are fallbacks if you don't have a hardware current sensor.  They generate estimates and approximations, but are not as accurate as measuring the current directly.
